x-pack/plugins/observability_solution/apm/public/components/app/transaction_details/waterfall_with_summary/waterfall_container/waterfall/waterfall_item.tsx

```diff
@@ -193,7 +193,7 @@ function NameLabel({ item }: { item: IWaterfallSpanOrTransaction }) {
   switch (item.docType) {
     case 'span':
       let name = item.doc.span.name;
-      if (item.doc.span.composite) {
+      if (item.doc.span.composite && item.doc.span.composite.count) {
```

Comment:
does it matter that this returns false when count is 0? probably not, but might be good to be more specific here.

Reply:
It was shown in the name so I had `undefined` in every span when I was testing - I don't think we should show `0` either but maybe what I faced was more of an edge case